geoips/commandline/geoips_list.py

```diff
@@ -183,12 +183,17 @@ def __call__(self, args):
             try:
                 listdir(unit_test_dir)
             except FileNotFoundError:
-                if len(package_names) == 1:
+                # I'm not sure why len(package_names) == 1 would indicate that the
+                # unit-tests folder does not exist.  Instead, check if the unit-tests
+                # folder exists to see if it exists.
+                # if len(package_names) == 1:
```

Comment:
This comment seems unnecessary. Remove? I understand why the comment was made, but the code is pretty self-explanatory and the comment fits better in a commit message.